README.md

```diff
@@ -1 +1,37 @@
-# andre-acosta-portfolio
\ No newline at end of file
+Andre Acosta Portfolio
+
+In this project I focus on building my personal portfolio to show off
+my best work!
+
+![Screen Shot 2022-01-05 at 17 24 49](https://user-images.githubusercontent.com/90656697/148305351-145b9ffc-6bfc-4e75-a228-e2bf3c9843d3.png)
+
+
+This Project has both a mobile and desktop version and is fully responsive
+to accommodate all screen sizes.
+
+Built With
+HTML5 & CSS3
+
+Technologies Used
+Hintrc
+Sylelint
+lighthouse
+
+
+Authors
+👤 Author1
+
+GitHub: @andreacostabz
+Twitter: @andrebze
+LinkedIn: Andre Acosta
```

Comment:
- [ ] You should consider adding your LinkedIn profile link instead of having simply your name.